backend/xivdpscalc/character/jobs.py

```diff
@@ -13,7 +14,9 @@ class Roles(Enum):
 
 
 class Buffs(Enum):
-    """ Buffs enum representing (magnitude, duration, cooldown) in seconds """
+    """
+    List of all buffs in the game. Each buff has three parameters: magnitude, duration, cooldown, in seconds.
+    """
     # aoe
     CHAIN = (0.1, 15, 120)
```

Comment:
Can we come up with a type for this?

Reply:
this breaks the program so i left it out